src/app/loadout-builder/filter/LockArmorAndPerks.tsx

```diff
@@ -123,6 +188,7 @@ export default memo(function LockArmorAndPerks({
           </ol>
         </div>
       )}
+      {/* Locked exotic */}
```

Comment:
Might be nicer to just make them components

Reply:
Yeah planning on doing this in another PR. This component is really big now.